packages/dart/sshnoports/bundles/core/config/sshnp-config-template.env

```diff
@@ -1,51 +0,0 @@
-### Template Config File for SSH No Ports ###
-## Entries do not require "" e.g
-## FROM=@alice
-
-# Sending atSign's atKeys file if not in ~/.atsign/keys/
-KEY_FILE=
-
-# Sending (a.k.a. client) atSign
-FROM=
-
-# Receiving (a.k.a. remote) device atSign
-TO=
```

Comment:
We don't really encourage people to use this, and it's not up to date, so let's just remove the template. People can still create config files if they want.